countryFlags.js

```diff
@@ -1,66 +1,231 @@
   "China": "🇨🇳",
   "Colombia": "🇨🇴",
+  "Comoros": "🇰🇲",
+  "Congo": "🇨🇬",
+  "Costa Rica": "🇨🇷",
+  "Croatia": "🇭🇷",
+  "Cuba": "🇨🇺",
+  "Cyprus": "🇨🇾",
   "Czech Republic": "🇨🇿",
+  "Czechia": "🇨🇿",
+  // D
+  "Democratic Republic of the Congo": "🇨🇩",
   "Denmark": "🇩🇰",
+  "Djibouti": "🇩🇯",
+  "Dominica": "🇩🇲",
+  "Dominican Republic": "🇩🇴",
+  // E
+  "Ecuador": "🇪🇨",
   "Egypt": "🇪🇬",
+  "El Salvador": "🇸🇻",
+  "Equatorial Guinea": "🇬🇶",
+  "Eritrea": "🇪🇷",
+  "Estonia": "🇪🇪",
+  "Eswatini": "🇸🇿",
+  "Ethiopia": "🇪🇹",
   "Europe": "🇪🇺",
+  "European Union": "🇪🇺",
+  // F
+  "Fiji": "🇫🇯",
   "Finland": "🇫🇮",
   "France": "🇫🇷",
+  // G
+  "Gabon": "🇬🇦",
+  "Gambia": "🇬🇲",
+  "Georgia": "🇬🇪",
   "Germany": "🇩🇪",
+  "Ghana": "🇬🇭",
   "Greece": "🇬🇷",
+  "Grenada": "🇬🇩",
+  "Guatemala": "🇬🇹",
+  "Guinea": "🇬🇳",
+  "Guinea-Bissau": "🇬🇼",
+  "Guyana": "🇬🇾",
+  // H
+  "Haiti": "🇭🇹",
+  "Honduras": "🇭🇳",
   "Hong Kong": "🇭🇰",
   "Hungary": "🇭🇺",
+  // I
+  "Iceland": "🇮🇸",
   "India": "🇮🇳",
   "Indonesia": "🇮🇩",
   "Iran": "🇮🇷",
   "Iraq": "🇮🇶",
   "Ireland": "🇮🇪",
   "Israel": "🇮🇱",
   "Italy": "🇮🇹",
+  "Ivory Coast": "🇨🇮",
+  // J
+  "Jamaica": "🇯🇲",
   "Japan": "🇯🇵",
+  "Jordan": "🇯🇴",
+  // K
+  "Kazakhstan": "🇰🇿",
   "Kenya": "🇰🇪",
+  "Kiribati": "🇰🇮",
+  "Korea": "🇰🇷",
+  "Kosovo": "🇽🇰",
+  "Kuwait": "🇰🇼",
+  "Kyrgyzstan": "🇰🇬",
+  // L
+  "Laos": "🇱🇦",
+  "Latvia": "🇱🇻",
+  "Lebanon": "🇱🇧",
+  "Lesotho": "🇱🇸",
+  "Liberia": "🇱🇷",
+  "Libya": "🇱🇾",
+  "Liechtenstein": "🇱🇮",
+  "Lithuania": "🇱🇹",
+  "Luxembourg": "🇱🇺",
+  // M
+  "Madagascar": "🇲🇬",
+  "Malawi": "🇲🇼",
   "Malaysia": "🇲🇾",
+  "Maldives": "🇲🇻",
+  "Mali": "🇲🇱",
+  "Malta": "🇲🇹",
+  "Marshall Islands": "🇲🇭",
+  "Mauritania": "🇲🇷",
+  "Mauritius": "🇲🇺",
   "Mexico": "🇲🇽",
+  "Micronesia": "🇫🇲",
+  "Moldova": "🇲🇩",
+  "Monaco": "🇲🇨",
+  "Mongolia": "🇲🇳",
+  "Montenegro": "🇲🇪",
+  "Morocco": "🇲🇦",
+  "Mozambique": "🇲🇿",
+  "Myanmar": "🇲🇲",
+  // N
+  "Namibia": "🇳🇦",
+  "Nauru": "🇳🇷",
+  "Nepal": "🇳🇵",
   "Netherlands": "🇳🇱",
   "New Zealand": "🇳🇿",
+  "Nicaragua": "🇳🇮",
+  "Niger": "🇳🇪",
   "Nigeria": "🇳🇬",
+  "North Korea": "🇰🇵",
+  "North Macedonia": "🇲🇰",
   "Norway": "🇳🇴",
+  // O
+  "Oman": "🇴🇲",
+  // P
   "Pakistan": "🇵🇰",
+  "Palau": "🇵🇼",
+  "Palestine": "🇵🇸",
+  "Panama": "🇵🇦",
+  "Papua New Guinea": "🇵🇬",
+  "Paraguay": "🇵🇾",
+  "Peru": "🇵🇪",
   "Philippines": "🇵🇭",
   "Poland": "🇵🇱",
   "Portugal": "🇵🇹",
+  // Q
+  "Qatar": "🇶🇦",
+  // R
   "Romania": "🇷🇴",
   "Russia": "🇷🇺",
+  "Rwanda": "🇷🇼",
+  // S
+  "Saint Kitts and Nevis": "🇰🇳",
+  "Saint Lucia": "🇱🇨",
+  "Saint Vincent and the Grenadines": "🇻🇨",
+  "Samoa": "🇼🇸",
+  "San Marino": "🇸🇲",
+  "Sao Tome and Principe": "🇸🇹",
   "Saudi Arabia": "🇸🇦",
+  "Senegal": "🇸🇳",
+  "Serbia": "🇷🇸",
+  "Seychelles": "🇸🇨",
+  "Sierra Leone": "🇸🇱",
   "Singapore": "🇸🇬",
+  "Slovakia": "🇸🇰",
+  "Slovenia": "🇸🇮",
+  "Solomon Islands": "🇸🇧",
+  "Somalia": "🇸🇴",
   "South Africa": "🇿🇦",
-  "Korea": "🇰🇷",
   "South Korea": "🇰🇷",
```

Comment:
from what I saw, `Korea` is the correct key, for South Korea

ref: https://x.com/KoreanTravel/about